autoscheduler/frontend/src/tests/ui/SchedulingPage.test.tsx

```diff
@@ -10,11 +10,33 @@ import { Provider } from 'react-redux';
 import {
   render, fireEvent, waitFor, queryByText as queryContainerByText,
 } from '@testing-library/react';
+import { navigate } from '@reach/router';
 import autoSchedulerReducer from '../../redux/reducer';
 import SchedulingPage from '../../components/SchedulingPage/SchedulingPage';
 import { mockFetchSchedulerGenerate } from '../testData';
 
+// Mocks navigate, so we can assert that it redirected to the correct url for Redirects to /schedule
+// This must be outside of all describes in order to function correctly
+jest.mock('@reach/router', () => ({
+  navigate: jest.fn(),
+}));
+
 describe('Scheduling Page UI', () => {
+  describe('redirects to the homepage', () => {
```

Comment:
Can you also add a test asserting that it doesn't redirect if a term is selected?

Reply:
![image](https://user-images.githubusercontent.com/49422137/95691665-f6217d80-0bdd-11eb-8cbd-f701cdd7b79c.png)
How's this?

Reply:
wait holdup. I just realized that this test is 100% empty, as in, nothing is actually rendered. How does that work?

Reply:
I think that when the fetchMock is reset it calls render which then leads to the redirect on the effectHook. If there's anything I assumed wrong let me know. Thanks!

Reply:
![image](https://user-images.githubusercontent.com/49422137/95794100-5f6bc400-0ca4-11eb-8082-4d9598a315fd.png)
![image](https://user-images.githubusercontent.com/49422137/95794067-4bc05d80-0ca4-11eb-8c81-c1165c3c06c6.png)
Do these look good?

Reply:
Yeah that looks good!

Just btw when you post code, it's easier to copy/paste it in and surround it with 3 grave (`)'s to make a [codeblock](https://guides.github.com/features/mastering-markdown//#GitHub-flavored-markdown). Just makes it easier to view/read on different devices.